public/content/documentation/web/component/password-input.md

```diff
@@ -115,10 +114,10 @@ Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/compo
 </example>
 
 ### Button variant
-   - The button leads the input so screen reader and keyboard-only users can change the state before interacting with the field.
-   - The password field type toggles between type of password and text.
-   - A live region role="status" is used to automatically announce to screen reader users the current visibility of the password value. For example, “Password is currently visible”.
-     - Avoid adding aria-hidden="true" or CSS display: none; to the live region container that receives the dynamic update as this may impact screen reader support.
+   - The show/hide button recieves focus before the input so screen reader and keyboard-only users can change the state before interacting with the field.
+   - The `password` field type toggles between type of password and `text`.
```

Comment:
The other "password" word in the sentence needs backticks instead of the one here; since "type" is referring to both "password" and "text" and the end of the sentence